### DISCUSSION TITLE
Consider making a box model control

### DISCUSSION THREAD

**Discussion by mor10** [GoogleChromeLabs/ProjectVisBug#191] [Ideas]
Consolidating the box model (margin, border, padding, width, height, etc) might make sense so all the controls related to a box are found in one place. having separate controls for each would require a long list of individual buttons and toggles.

**Comment by argyleink** [comment]
i believe this is related to #97, but what i like about your request is putting things under the umbrella of the box model. do you think 97 covers what you're asking for in this issue? or is this issue more specifically about being able to toggle between border-box and content-box models?

**Comment by JohnAkerman** [comment]
I'm wondering whether this could be extended to include z-index to allow for easier stacking order changes.

**Comment by argyleink** [comment]
hmmm, z-index would work nicely as part of the position tool? it hardly has any keyboard controls at the moment. want to log a separate bug about z-index? also.. lol, it's really only helpful to people who *dont* use values like 9999... that's be a loooot of keyboard pressing.

**Comment by argyleink** [comment]
i could see a box model control just like chrome has today being part of the 'design tab' or '2nd screen experience' panels that #205 is trying to do? maybe i mistook the original intent of this issue earlier, and what it really wanted was the box model visual and feedback that you get from the devtools today. i do believe designers have found that useful.
